7_deploy_osh/roles/suse-osh-deploy/tasks/apply_on_k8s.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: {{ item.action }} - file generation
```

Comment:
This needs to be quoted

Reply:
Yes, and user doesn't exist... It was done late yesterday :P